openupgradelib/openupgrade.py

```diff
@@ -909,20 +909,23 @@ def write_value(ids, field, value):
                 write_value(ids, field, value)
 
 
-def logged_query(cr, query, args=None):
+def logged_query(cr, query, args=None, skip_no_result=False):
     """
     Logs query and affected rows at level DEBUG.
 
     :param query: a query string suitable to pass to cursor.execute()
-    :param args: a list, tuple or dictionary passed as substitution values \
-to cursor.execute().
+    :param args: a list, tuple or dictionary passed as substitution values
+      to cursor.execute().
+    :param skip_no_result: If True, then logging details are only shown
```

Comment:
Good addition. But should not be True by default. I mean, a request without result has no interest. don't you think ?

Reply:
And it's not... This is the documentation.

Reply:
OK, we'll talk about that point in another thread.

Reply:
It's not needed, please check the keyword argument. It's by default `False`.

Reply:
Yep. I checked. I didn't explain my PoV correctly.

I just wanna say that by default, **it should be True** to only show relevant log. (but it changes the behaviour of the function, so let's go talking about that point on another thread)

Reply:
Ah, got it! Yeah, I was conservative with previous behavior. We can discuss to see if we prefer to not show them by default.

Reply:
I think this could be a global setting (Odoo's debug mode?) so that developers can see all queries while at runtime you only see the effective queries.